pyfstat/tcw_fstat_map_funcs.py

```diff
@@ -266,6 +266,16 @@ def _optional_imports_pycuda():
     )
 
 
+def _get_transient_fstat_map_features():
+    """Helper function to check available features."""
+    features = {}
+    have_lal = _optional_import("lal")
+    have_lalpulsar = _optional_import("lalpulsar")
+    features["lal"] = have_lal and have_lalpulsar
```

Comment:
For the record, I confirm that `_optional_import` returns a `bool` and hence `features["lal"]` will *always* be a boolean as constructed here.